docs/user/exams/instructors_guide.rst

```diff
@@ -0,0 +1,192 @@
+
+     *Exam Management* -> *Exercise Groups* -> *View* on the specific exercise.
+
+     At the bottom of the page you will find the option ``check for plagiarism``.
+
+.. figure:: instructor/tutor_dashboard.png
+   :alt: Tutor Exam Dashboard
+   :align: center
+
+   Tutor Exam Dashboard
+
+- To assess a submission for an exercise, you can click on ``Exercise Dashboard``.
+- First you must go through the example submissions and assessments to review how a specific exercise should be evaluated. 
+- If there is a submission which has not been assessed yet, you can click ``Start new assessment``. This will fetch a random student submission of this exercise which you can then assess.
+- Programming exercises are graded automatically but if ``manual assessment`` is allowed, see `Add Exercises`_, you can review and enhance the automatic results. 
+
+.. figure:: instructor/programming_assessment.png
+   :alt: Programming Submission Assessment
+   :align: center
+
+   Manually Assessing a Programming Submission
+
+- Quiz exercises are graded automatically via the student exam page, see `Manage Student Exams`_, and therefore do not appear in the *Tutor Exam Dashboard*.
+- Students can complain about their results during the :ref:`student review <student_review>` period. You can evaluate these complaints also in the *Tutor Exam Dashboard*. 
+- The original assessor may not respond to the complaint about his assessment, this must be done by a second assessor. 
+
+Exam Scores
+^^^^^^^^^^^
+- You can view the exam scores from the *Scores* page. This view aggregates the results of the students and combines them to provide an overview over the students’ performance. 
+- You can view the spread between different achieved scores, the average results per exercise as well as the individual students' results.
+- Additionally, you can choose to modify the dataset by selecting ``only include submitted exams`` or ``only include exercises with at least one non-empty submission``.
+- The exam scores can also be exported via ``Export Results as CSV``.
+
+.. figure:: instructor/exam_statistics.png
+   :alt: Exam Scores page
+   :align: center
+
+   Exam Scores Page
+
+Conducting Test Runs
+^^^^^^^^^^^^^^^^^^^^
+- Test runs are designed to offer the instructors confidence that the :ref:`exam conduction <exam_conduction>` will run smoothly. They allow you to experience the exam from the student’s perspective. A ``test run`` is distinct from a ``student exam`` and is not taken into consideration during the calculation of the exam scores. 
+- You can manage your test runs from the *Test Run* page.
+- To create a new test run you can press ``Create a Test Run``. This will open a modal where you can select an exercise for each exercise group. You can also set the ``working time``. A test run will have as many exercises, as there are exercise groups. It does not take the ``number of exercises`` set in the exam configuration under consideration.
```

Comment:
please add some images here as well.
1) Create test run
2) Participate in test run
3) Manage (in particular delete) test runs